From d11ffef6d8a68cb9750bd88eb407814268e9747a Mon Sep 17 00:00:00 2001
From: Dinesh Karthikeyan <quic_dinek@quicinc.com>
Date: Wed, 10 Aug 2022 19:03:37 +0530
Subject: [PATCH] ath12k: Change default frame mode

Set ATH12K_HW_TXRX_ETHERNET mode as default frame_mode in driver.

Signed-off-by: Dinesh Karthikeyan <quic_dinek@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/core.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/net/wireless/ath/ath12k/core.c b/drivers/net/wireless/ath/ath12k/core.c
index 4371655..3165f03 100644
--- a/drivers/net/wireless/ath/ath12k/core.c
+++ b/drivers/net/wireless/ath/ath12k/core.c
@@ -26,7 +26,7 @@ module_param_named(crypto_mode, ath12k_crypto_mode, uint, 0644);
 MODULE_PARM_DESC(crypto_mode, "crypto mode: 0-hardware, 1-software");
 
 /* frame mode values are mapped as per enum ath12k_hw_txrx_mode */
-unsigned int ath12k_frame_mode = ATH12K_HW_TXRX_NATIVE_WIFI;
+unsigned int ath12k_frame_mode = ATH12K_HW_TXRX_ETHERNET;
 module_param_named(frame_mode, ath12k_frame_mode, uint, 0644);
 MODULE_PARM_DESC(frame_mode,
 		 "Datapath frame mode (0: raw, 1: native wifi (default), 2: ethernet)");
 -- 
2.17.1

